qiskit_experiments/characterization/t1_experiment.py

```diff
@@ -26,7 +26,7 @@
 from qiskit_experiments.analysis.curve_fitting import process_curve_data, curve_fit
 from qiskit_experiments.analysis.data_processing import level2_probability
 from qiskit_experiments.analysis import plotting
-from qiskit_experiments import AnalysisResult
+from qiskit_experiments.experiment_data import AnalysisResult
```

Comment:
Why this is updated?

Reply:
It was giving me cyclical imports when running lint locally (always has, but the CI does not seem to pick-up on it). Also importing `AnalysisResults` from `qiskit_experiments` and not `qiskit_experiments.experiment_data` does not seem to match the way we import in Qiskit experiments.